kdbg/brkpt.cpp

```diff
@@ -6,7 +6,6 @@
 
 #include <klocale.h>			/* i18n */
 #include <kiconloader.h>
-#include <ksimpleconfig.h>
```

Comment:
This looks like an independent change. Can you please split it off into a separate commit? It would be nice to know what "recent" KDE frameworks means. (I guess it is all of KF5.)